docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.8'
```

Comment:
Could you please add a bried description and command to run the docker file in README?

Reply:
Added!